src/lib/cql-execution/example/cql/PALLASEligibility.cql

```diff
@@ -35,7 +35,7 @@ concept "Progesterone Receptor": { "Progesterone receptor [Interpretation] in Ti
 concept "Hemoglobin": { "Hemoglobin A1c [Mass/volume] in Blood" } display 'Hemoglobin A1c'
 concept "Platelets": { "Platelets [#/volume] in Blood" }  display 'Platelets [#/volume] in Blood'
 concept "HER2": { "HER2 panel - Tissue by FISH" } display 'HER2 panel - Tissue by FISH'
-concept "AJCC": { "AJCC cancer staging manual [PhenX]" } display 'AJCC cancer staging manual'
+concept "Staging": { "AJCC cancer staging manual [PhenX]" } display 'Cancer staging manual'
```

Comment:
Do we want this reference to AJCC Removed?

Reply:
That's the canonical display text for a specific LOINC code. I had suggested offline that in that case it made more sense to leave the canonical text rather than change it

Reply:
Exactly right. Since all we're doing is referencing a LOINC code, and since LOINC's licensing is generous, this is a problem b/t LOINC and AJCC, not us 👍